mantidproject/vesuvio
# Switch tests directories

**Description of work:**
Change structure of tests from `tests/calibration/unit' to 'tests/unit/calibration/'
This makes it simpler to run all unit tests with a single path

**To test:**
Check that all tests in last commit were all run and completed successfully




Fixes #xxxx.
